lein-eftest/src/leiningen/eftest.clj

```diff
@@ -16,10 +16,10 @@
                              "function!")
                         {:reporter reporter}
                         cause))))
-    'eftest.report.progress/report))
+    'eftest.report.progress))
 
 (defn- require-form [project]
-  `(require 'eftest.runner '~(report-namespace project)))
+  `(require 'eftest.report 'eftest.runner '~(report-namespace project)))
```

Comment:
We might not _need_ the report namespace, but given the relatively low cost of requiring it I opted for keeping things simple.